platform/darwin/src/MGLRendererConfiguration.mm

```diff
@@ -1,19 +1,46 @@
 #import "MGLRendererConfiguration.h"
 #import "MGLOfflineStorage_Private.h"
+#import "MGLFoundation_Private.h"
 
 #if TARGET_OS_IPHONE
 #import <UIKit/UIKit.h>
 #else
 #import <AppKit/AppKit.h>
 #endif
 
+static NSString * const MGLCollisionBehaviorPre4_0Key = @"MGLCollisionBehaviorPre4_0";
 
 @implementation MGLRendererConfiguration
 
 + (instancetype)currentConfiguration {
     return [[self alloc] init];
 }
 
+- (instancetype)init {
+    return [self initWithPropertyDictionary:[[NSBundle mainBundle] infoDictionary]];
+}
+
+- (instancetype)initWithPropertyDictionary:(NSDictionary *)properties {
+    
+    if (self) {
```

Comment:
This is missing a call to `-[NSObject init]`.

Reply:
Whoops! Lost that in the re-shuffle.